modules/swagger-codegen/src/main/resources/swift3/Configuration.mustache

```diff
@@ -0,0 +1,11 @@
+// Configuration.swift
+//
+// Generated by swagger-codegen
+// https://github.com/swagger-api/swagger-codegen
+//
+
+import Foundation
+
+open class Configuration {
+    open static var dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"
```

Comment:
I understand that changing the value after the constructor will have no effect. I can fix that if you want. 

However, as for the necessity, I believe it is necessary. The auto generated client code must not dictate the server code to any date format. If you don't want to change the client code, I can add it as a command line parameter. What do you think?

C# client has this flexibility. 
https://github.com/swagger-api/swagger-codegen/blob/master/samples/client/petstore/csharp/SwaggerClient/src/IO.Swagger/Client/Configuration.cs#L284-L315